### PR TITLE
Disabling jog for each deck separately

### PR DESCRIPTION
**Pull request by fkbreitl**

### CODE REVIEW

File: Pioneer-DDJ-200-scripts.js [hunk 1]
```diff
@@ -25,11 +25,12 @@ DDJ200.jog = function (channel, control, value, status, group) {
 DDJ200.touch = function (channel, control, value, status, group) {
     var deckNumber = script.deckFromGroup(group);
     if (value == 0) {
+        // disable jog to not prevent alignment and enable it after 900 ms
+        DDJ200.jog_disabled[deckNumber] = true;
```
Comment (dan-giddins) [inline Pioneer-DDJ-200-scripts.js]: you will also need to check jog_disabled based on deck number

**Review by dan-giddins — CHANGES_REQUESTED** [review]
you will also need to check jog_disabled based on deck number

**Review by fkbreitl — COMMENTED** [review]
Yes of course. Thanks for the correction.
Sorry, mistakes happen very easily because they still have not executed our PR and I have to switch back and forth between branches.
I am already working on the 4D (4Decks) branch.
You can have a look. However, I am still struggling with getting the volume mapped in:
https://mixxx.discourse.group/t/mapping-volume-fader-to-javascript-function/20104/
Cheers!

**Review by fkbreitl — COMMENTED** [review]
fixed jog_disabled[deckNumber]

**Comment by fkbreitl** [comment]
Hi @Dan, the mapping should be polished now for including in Mixxx. Can you pull it?

**Review by dan-giddins — APPROVED** [review]

**Pull request merged**